library.js

```diff
@@ -58,6 +58,7 @@ const plugin = {
     noRegistration: 'off',
     payloadParent: undefined,
     allowBannedUsers: false,
+    updateProfile: 'on',
```

Comment:
Brukes denne settingen noe sted? Eller hvordan fungerer den?

Reply:
Sjekk for å kun kjøre når den innstillingen er skrudd på